spec/lib/chat_statistics_spec.rb

```diff
@@ -0,0 +1,88 @@
+# frozen_string_literal: true
+
+describe DiscourseChat::Statistics do
+  fab!(:frozen_time) { DateTime.parse("2022-07-08 09:30:00") }
+
+  def minus_time(time)
```

Comment:
This weirdness has to be done since `freeze_time` does not work with `fab!` , the `before do` blocks run after the `fab!` calls